server/lib/payments.js

```diff
@@ -393,9 +393,13 @@ export async function associateTransactionRefundId(transaction, refund, data) {
 
 export const sendEmailNotifications = (order, transaction) => {
   debug('sendEmailNotifications');
-  // for gift cards and manual payment methods
-  // Don't send a "thank you" email when moving funds between a sub-collective and its parent
-  if (transaction && order.fromCollective?.ParentCollectiveId !== order.collective?.id) {
+  // Don't send a "thank you" email when moving funds between a sub-collective and its parent or between host
+  // and one of its collectives.
```

Comment:
So, it's a DEBIT or a CREDIT we're looking at? Assuming DEBIT.

So, case 1) let's say order.fromCollective (the recipient) = {slug: sustainoss-2020, type: event, parentCollective = sustainoss} and order.collective (the sender) = {slug: sustainoss, type: collective}. That would work.

So, case 2) let's say order.fromCollective (the recipient) = {slug: sustainoss, type: collective, hostCollective = opensource} and order.collective (the sender) = {slug: opensource, type: organization}. That doesn't work.

There is something wrongs it seems. Maybe by explaining this in comments the cases we would be able to read the code and make sure it's right?

By the way, have you tested the code?

Reply:
In your cases is the recipient and sender correct? My understanding is that, `order.fromCollective` should be the sender whereas the `order.collective` should be the recipient? 🤔 

Yes I have tested this. I think here what we are trying to fix is the contribution from host to one of it's collective part (the second point you mentioned). This one I tested with a org profile (host) contributing to one of it's collectives. Not sure if I missed anything, but it didn't work for you? I've just changed the comment to make it a bit more readable and to show from where to where the funds are going. 🤔

Reply:
> So, it's a DEBIT or a CREDIT we're looking at? Assuming DEBIT.

Could you answer this question? I'm not working on the issue and, without this, reviewing is difficult. Can you make sure to double check for case 1) and case 2) and tell us what it is? Can you also add the information in the code comments?

Reply:
So I checked the transaction types and they are CREDIT for all cases, 

1) Child to Parent Collective: CREDIT
2) Host to its own collective: CREDIT
3) Parent Collective to Child: CREDIT
4) Collective to its Host: CREDIT

I've added some code comments, and also made sure to support the back transfers from the parent collective to the child and from the collective to its host. 

>Why would we be checking order.fromCollective?.ParentCollectiveId if order.fromCollective is the sender? We're lost and that's a problem. Let's make things clear.

This check in particular for the case where we transfer from the child collective (event/project) to it's parent. I've also added the condition `order.fromCollective?.id !== order.collective?.ParentCollectiveId` to support the case where the parent collective transfers to the child collective. 

Let me know if things look better now. 🤔